src/main/resources/application.properties

```diff
@@ -7,4 +7,7 @@ spring.datasource.password=password
 spring.main.banner-mode=off
 
 # Hibernate
-spring.jpa.hibernate.ddl-auto=update
\ No newline at end of file
+spring.jpa.hibernate.ddl-auto=update
+
+# Remove trace from ResponseStatusException
+server.error.include-stacktrace=never
```

Comment:
Ahhhh your first production specific variable. Don't worry about this for now but soon you going to need to look into how to have 2 types of configurations . . . production vs development. Just prepping you for that.